MFVNeutralino/plugins/JetTreer.cc

```diff
@@ -244,7 +244,7 @@ void MFVJetTreer::analyze(const edm::Event& event, const edm::EventSetup&) {
   double min_track_nstlayers = 6;
   double min_track_hit_r = 1;
   
-  const int track_rescaler_which = jmt::TrackRescaler::w_JetHT; // JMTBAD which rescaling if ever a different one
+  const int track_rescaler_which = jmt::TrackRescaler::w_BTagDispJet; // JMTBAD which rescaling if ever a different one
```

Comment:
I think this should become a configurable option, if we are using different rescaling values for bjets vs. VH vs. displaced leptons. Otherwise everyone has to go remember to change their hardcoded values, and we'll never get it correct.